src/utils/APIBreaks/APIBreak.ts

```diff
@@ -153,44 +153,45 @@ export function showAPIBreaks(version: string, { additionalText, apiBreakingChan
   }
   Player.getHomeComputer().writeToTextFile(
     textFileName,
-    `API BREAK INFO FOR ${version}\n\n${details.map((detail) => detail.text).join("\n\n\n\n")}`,
+    `API BREAK INFO FOR ${version}\n\n${details.map((detail) => detail.text).join("\n\n\n\n")} \n\n${additionalText}`,
   );
 
   Terminal.warn(`AN API BREAK FROM VERSION ${version} MAY HAVE AFFECTED SOME OF YOUR SCRIPTS.`);
   Terminal.warn(`INFORMATION ABOUT THIS POTENTIAL IMPACT HAS BEEN LOGGED IN ${textFileName} ON YOUR HOME COMPUTER.`);
 
+  
+  // Show a dialog box with the API break info
+  
   dialogBoxCreate(
-    `SOME OF YOUR SCRIPTS HAVE POTENTIALLY BEEN IMPACTED BY AN API BREAK, DUE TO CHANGES IN VERSION ${version}\n\n` +
-      "The following dialog boxes will provide details of the potential impact to your scripts.\n" +
-      `A file with these details has also been saved on your home computer under filename ${textFileName}.` +
-      (additionalText ? `\n\n${additionalText}` : ""),
+    `SOME OF YOUR SCRIPTS HAVE POTENTIALLY BEEN IMPACTED BY AN API BREAK, DUE TO CHANGES IN VERSION ${version}` +
+      (additionalText ? `\n${additionalText}` : "") +
+      `\n\nPlease check the log file ${textFileName} on your home computer for more details.` +
+      `\n${details.map((detail) => detail.apiBreakInfo.info).join("\n\n\n\n")}`,
   );
-  let popUpIndex = 0;
+
+  /**let popUpIndex = 0;
   for (const detail of details) {
     if (!detail.showPopUp) {
       continue;
     }
-    dialogBoxCreate(
-      `API BREAK VERSION ${version} DETAILS ${popUpIndex + 1} of ${numberOfPopUps}\n\n${detail.apiBreakInfo.info}` +
-        /**
-         * If we can detect the affected lines via apiBreakInfo.brokenAPIs, we will show the number of affected lines.
-         * However, some breaking changes cannot be reliably detected, so we intentionally leave apiBreakInfo.brokenAPIs
-         * empty. With these changes, the number of affected lines is always 0, but saying that there are no affected
-         * lines is misleading, so we won't say anything about the number of affected lines.
-         */
-        (detail.apiBreakInfo.brokenAPIs.length > 0
-          ? `\n\nWe found ${pluralize(detail.totalDetectedLines, "affected line")}.`
-          : ""),
-    );
+    if (popUpIndex >= numberOfPopUps) {
+      break;
+    }
+  */
 
-    ++popUpIndex;
-  }
   //Made terminal more verbose
-  for (const detail of details) {
-    if (detail.totalDetectedLines > 0) {
-      Terminal.warn("_____________");
+  for (const detail of details) { 
```

Comment:
Looking at this, and thinking through the consequences, I realize that switching to writing the details in the terminal will actually be rather involved, with only the smallest part of it being showing the details in color. (There's stuff like renaming fields where "showPopup" isn't correct anymore.)

So, I'm going to suggest that we cut that from this PR to avoid score creep, and do it in a different PR (possibly in parallel) that can focus solely on that.